### PR TITLE
Added unit test coverage capability

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: npm/build-docs.js [hunk 1]
```diff
@@ -5,19 +5,20 @@ var path = require('path'),
...
+    INFO_MESSAGE = 'Generating documentation...'.yellow.bold,
     FAILURE_MESSAGE = 'Documentation could not be generated!'.red.bold,
+    SUCCESS_MESSAGE = ' - documentation can be found at ./out/docs'.green.bold,
     DOC_GENERATION_COMMAND = (IS_WINDOWS ? '' : 'node ') + path.join('node_modules', '.bin', 'jsdoc') +
         (IS_WINDOWS ? '.cmd' : '') + ' -c .jsdoc-config.json -u docs lib';
 
-module.exports = function (exit) {
-    console.log('Generating documentation...'.yellow.bold);
+module.exports = function (done) {
+    console.log(INFO_MESSAGE);
```
Comment (czardoz) [inline npm/build-docs.js]: What does this have to do with Istanbul?

File: npm/test-system.js [hunk 2]
```diff
@@ -77,7 +78,7 @@ module.exports = function (exit) {
                 return next();
             });
         }
-    ], exit);
+    ], done);
```
Comment (czardoz) [inline npm/test-system.js]: These changes make the scripts different from the runtime scripts. Why have you renamed `exit` to `done`?

Comment (kunagpal) [inline npm/test-system.js]: The scripts are now less confusing, with unique names for the context-specific callback.

File: npm/build-docs.js (hunk 1 above)
Comment (kunagpal) [inline npm/build-docs.js]: Nothing, but the string message sources are now standardised.
